base/linalg/triangular.jl

```diff
@@ -1440,6 +1623,7 @@ function logm{T<:Union{Float64,Complex{Float64}}}(A0::UpperTriangular{T})
             end
             if j <= 6
                 m = j
+                foundm = true
```

Comment:
Since `foundm` is not really used in this loop and afterwards, I'm tempting to suggest to eliminate it completely, e.g.:
```julia
if alpha2 <= theta[2]
    m = alpha2 <= theta[1] ? 1 : 2
else
    while true
       <...>
    end
end
```
We would save a few lines + improve (IMHO) the clarity at the expense of indentation.